extensions/wikia/ContentReview/controllers/templates/ContentReviewModule.mustache

```diff
@@ -12,11 +12,9 @@
 
 	{{#displaySubmit}}
 	<div class="content-review-module-submit-container">
-		<a href="#" id="content-review-module-submit" title="{{submit}}">
-		<button class="content-review-module-button primary" type="button">
+		<button class="content-review-module-submit primary" data-page-name="{{{pageName}}}">
```

Comment:
This one is still vulnerable to XSS. Always default to two braces in Mustache unless you know it's already escaped.